lichess-bot.py

```diff
@@ -132,19 +134,44 @@ def logging_configurer(level: int, filename: Optional[str]) -> None:
         file_handler.setFormatter(file_formatter)
         all_handlers.append(file_handler)
 
+    if auto_log_filename:
+        if not os.path.isdir("./log/"):
+            os.mkdir("./log/")
```

Comment:
This `if` block can be replaced by a call to `os.makedirs("./log/", exist_ok=True)`

https://docs.python.org/3/library/os.html#os.makedirs

Reply:
Also, I think the name `log` is likely to conflict with existing directories that user's may use to store logs. A name like `lichess_bot_weekly_logs` is less likely to collide with user data.

Reply:
Also, also, the log directory can be derived from `auto_log_file_name` with `os.path.dirname()`

https://docs.python.org/3/library/os.path.html#os.path.dirname